scm-webapp/src/main/java/sonia/scm/api/v2/resources/AuthenticationRequestDto.java

```diff
@@ -32,8 +32,8 @@
 
 public class AuthenticationRequestDto {
 
-  @FormParam("grant_type")
-  @JsonProperty("grant_type")
+  @FormParam("grantType")
+  @JsonProperty("grantType")
   private String grantType;
```

Comment:
This is a breaking rest api change which is not allowed!

Reply:
Regardless of the breaking change, we use a underscore to align with the oauth spec.